src/engine/renderer/tr_local.h

```diff
@@ -597,6 +597,7 @@ static inline void halfToFloat( const f16vec4_t in, vec4_t out )
 
 		int            frameUsed; // for texture usage in frame statistics
 
+		int            formatSizeFactor;
```

Comment:
unused

Reply:
ah yes, a leftover from work-in-progress implementation. I noticed storing the result of division as an int was bad about precision XD.